contracts/HashRegistrarSimplified.sol

```diff
@@ -132,8 +133,11 @@ contract Registrar {
     //   Owned -> Open (releaseDeed)
     function state(bytes32 _hash) constant returns (Mode) {
         var entry = _entries[_hash];
-        if(now < entry.registrationDate) {
-            if(now < entry.registrationDate - revealPeriod) {
+        
+        if(!isAllowed(_hash, now)) {
+            return Mode.Unavailable;
```

Comment:
Can we reuse `Mode.Forbidden` here? I don't think anything will treat the two differently.